src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -23,7 +23,7 @@
 FAQ_MESSAGE = "\nPlease refer to https://aka.ms/acr/health-check for more information."
 ERROR_MSG_DEEP_LINK = "\nPlease refer to https://aka.ms/acr/errors#{} for more information."
 MIN_HELM_VERSION = "2.11.0"
-HELM_VERSION_REGEX = re.compile(r'SemVer:"v([.\d]+)"', re.I)
```

Comment:
By removing `re.I`, we can enforce a case sensitive regex match. (SemVer & Version)